src/ProjectNeon/Assets/Features/Effects/InterceptAttack.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+class InterceptAttack : Effect
+{
+    private Member _performer;
+    private Target _effectTarget;
+
+    void Effect.Apply(Member source, Target target)
+    {
+        _performer = source;
+        _effectTarget = target;
+        BattleEvent.Subscribe<Attack>(attack => Execute(attack), this);
```

Comment:
This must be cleaned up.

This effect is supposed to expire at the end of the turn.